plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/impls/redshift/PostgreServerRedshift.java

```diff
@@ -83,7 +83,193 @@ public PostgreServerRedshift(PostgreDataSource dataSource) {
         "TEMP",
         "UNLOAD",
         "VACUUM",
-        "YES"
+        "YES",
+        "AES128",
+        "AES256",
+        "ALL",
+        "ALLOWOVERWRITE",
+        "AES128",
```

Comment:
This is a duplicate. Please remove it and

```
"AES256",
"ALL",
"ALLOWOVERWRITE",
```

Reply:
removed it